### PR TITLE
Update scaling out readme section to reflect the current status

### PR DESCRIPTION
**Pull request by a1exsh**
The EBS volume should be created and tagged before launching new instance.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -285,26 +285,29 @@ following these steps:
...
+
+#. Create a ``Name`` tag on the volume in the format:
+   ``<cluster-name>-<private-ip>``.
+
 #. Use the 'Launch More Like This' menu in the AWS web console on one
    of the running nodes.
 
-#. If present remove the volumes section from user data. It references
-   the data volume of the original instance.
-
```
Comment (SergioGonzalezSanz) [inline README.rst]: The data volume in the user data must be updated to match the IP of the new node

Comment (a1exsh) [reply README.rst]: Mentioned below: https://github.com/zalando-stups/planb-cassandra/pull/187/files#diff-88b99bb28683bd5b7e3a204826ead112R308

Comment (SergioGonzalezSanz) [reply README.rst]: True, I missed it. Thanks!

**Comment by a1exsh** [comment]
:+1:

**Comment by slitsche** [comment]
👍

**Pull request merged**